src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -421,8 +420,8 @@
   "com.minecolonies.command.loadbackup.success": "Successfully loaded backup.",
   "com.minecolonies.command.export.success": "Exported colony to zip: %s",
   "com.minecolonies.command.deleteable.success": "Changed deletable flag of colony ID %s. It is now set to %s.",
-  "com.minecolonies.command.rsreset.success": "The request system for colony %s has been restarted in 1.618 seconds.",
-  "com.minecolonies.command.rsresetall.success": "The request systems for all colonies have been restarted in 1.618 seconds.",
```

Comment:
^Please leave this in, it is an inside joke. It is not ment to be a measure at all.